src/content/api/payment-requests.mdx

```diff
@@ -572,6 +572,17 @@ Payment Activities are created when a Payment Request has been `created`, `paid`
 
 ---
 
+<Endpoint
+  path="/api/payment-requests/{paymentRequestId}/summary"
+  filename="payment-requests-get-summary"
+>
+  ## Get Payment Request Summary
+
+  This endpoint allows you to retrieve the summary of a Payment Request while the status is `new`.
```

Comment:
I think you should also say somewhere in the description that you don't need to be authenticated to call this endpoint.

Reply:
Yeah was wondering about that - I'm not sure why the asset summary endpoints don't say anything. I guess the example request shows that you don't need an auth token but it still isn't that clear.

Do you think a `public` badge or something similar would be a good option?

Reply:
Yes, a public badge seems like a good idea.

Reply:
I've added a followup task for this as there is no design for this yet https://www.notion.so/centrapay/Add-public-badge-to-all-public-routes-in-Centrapay-docs-69c09d32b8154530a6974a66859f2497?pvs=4